andela/haven-ah-frontend
# #162234039 Display featured author on the home page

#### What does this PR do?
Create a section on the homepage to display the author of the week.

#### Description of Tasks to be completed
- write actions, reducers and sagas to handle API requests and update the store
- write featured author component

#### How can this be manually tested?
- on the terminal run `git clone https://github.com/andela/haven-ah-frontend.git`.
- run `git checkout ft-author_of_the_week-162234039`.
- run `npm install` to install the npm packages.
- run `npm run start:dev` to start the server. The homepage should open up on `localhost:8080`.
#### Screenshots if applicable
<img width="1271" alt="screen shot 2018-12-04 at 2 02 31 pm" src="https://user-images.githubusercontent.com/29674347/49449175-a18b6b00-f7da-11e8-99b0-a79d4107049f.png">




#### Relevant Pivotal Tracker stories
#162234039
